woocommerce/woocommerce-ios
# Move Hardware from a Swift Package to a subproject

Closes #3681 
For context, see: paaHJt-1gq#comment-3828-p2

The original setup (where `>` indicates a dependency) was:
WooCommerce > Hardware > Stripe Terminal SDK

It turns out there seems to be a limitation in the Swift Package Manager that prevents embedded frameworks from being codesigned, which in turn was making the app impossible to run on devices.

This PR migrates Hardware from a Swift Package to a subproject (like Yosemite or Networking) that builds a framework.

Also, the Stripe Terminal SDK is pulled in via Cocoapods

## Changes
* Removed the Hardware Swift Package
* Add a new subproject, called Hardware
* Move the code in the Hardware Swift Package to the new Hardware Subproject
* Add the Stripe Terminal SDK to the Podfile.
* Add a new section to the Podfile to setup the Hardware subproject

## Testing
* Checkout the branch, run `bundle exec pod install`
* Build and run. Make sure the project runs as expected.
* Add a breakpoint to line  of `HardwareTests.swift`. Run all the WooCommerce tests (i.e. press command+u) and make sure that the breakpoint in HardwareTests is hit. 
* Build and run on a device. The app should run on a device properly

Update release notes:
